doc/source/parameters/cfd/linear_solver_control.rst

```diff
@@ -0,0 +1,157 @@
+
+	  # AMG w cycling. If this is set to true, W cycling is used. Otherwise, V cycling is used.
+	  set amg w cycles              = false
+
+	  # AMG smoother sweeps
+	  set amg smoother sweeps       = 2
+
+	  # amg smoother overlap
+	  set amg smoother overlap      = 1
+	end
+
+
+* The ``method`` parameter enables to choose an iterative solver for the linear system of equations. Lethe currently supports these core solution strategies :
+	* ``gmres`` (default parameter value), a GMRES iterative solver with ILU preconditioning.
+	* ``amg``, a GMRES iterative solver with AMG preconditioning and an ILU coarsener and smoother.
+	* ``bicgstab``, a BICGSTAB iterative solver with ILU preconditioning.
+	* ``direct``, a direct solver using `TrilinosWrappers::SolverDirect <https://www.dealii.org/current/doxygen/deal.II/classTrilinosWrappers_1_1SolverDirect.html>`_. 
+
+	.. tip:: 
+		On coarse meshes, the ``gmres``/``bicgstab`` solver with ILU preconditioning is more efficient. 
+
+		As the number of mesh elements increase, the ``amg`` solver is the most efficient. Generally, at 1M elements, the ``amg`` solver always outperforms the ``gmres`` or ``bicgstab`` for 2D steady-state problems.
+		
+		The use of ``direct`` solver should be avoided for 3D problems.
+
+* The ``verbosity`` option enables the display of the residual at each non-linear iteration, to monitor the progress of the linear iterations.
+
+.. note::
+	Example of a ``set verbosity = verbose`` output:
+
+	.. code-block:: text
+
+		-Tolerance of iterative solver is : 0.0429541
+		-Iterative solver took : 11 steps 
+		-Tolerance of iterative solver is : 3.62082e-05
+		-Iterative solver took : 16 steps 
+		-Tolerance of iterative solver is : 1.05775e-08
+		-Iterative solver took : 17 steps 
+		-Tolerance of iterative solver is : 1.00205e-12
+		-Iterative solver took : 16 steps 
+		-Tolerance of iterative solver is : 1e-13
+		-Iterative solver took : 5 steps 
+		-Tolerance of iterative solver is : 1e-13
+		-Iterative solver took : 0 steps 
+
+
+* The ``minimum residual`` parameter sets the value that the tolerance of the iterative solver should reach to proceed to the next iteration.
+.. tip::
+	The ``minimum residual`` parameter is directly linked to the numerical convergence of the simulation, but also to the computational cost. 
+
+	A good rule of thumb is to set the linear solver ``minimum residual`` to one or two order of magnitude below the `Non-linear solver <https://lethe-cfd.github.io/lethe/parameters/cfd/non-linear_solver_control.html>`_ ``tolerance`` parameter (linear ``minimum residual`` :math:`=` non-linear ``tolerance`` :math:`/10` or :math:`/100`).
+
+	Typical values of the ``minimum residual`` for simple simulations can be quite low, for instance ``set minimum residual = 1e-13``. However, such a tolerance can be impossible to attain for more complex simulations, and so the ``minimum residual`` can be increased, for instance ``set minimum residual = 1e-5``.
```

Comment:
I would not necessarily add this comment. The best way to control is generally to have a relative residual that is not too small instead. I would just suggest to always set it at least 10x (pref 100x) smaller than the non-linear tolerance and to keep the relative residual reasonable (I find 1e-3 is a good value).